daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/EngineTest.scala

```diff
@@ -180,278 +180,6 @@ class EngineTest
   }
 
   "command translation" should {
-    "translate create commands argument including labels" in {
```

Comment:
The delete test are cover by the one added in ValueTransalator (this PR) and the one of CmmandPreprocessorSpec (added in #10909)